guides/cookbook.md

```diff
@@ -771,6 +1109,10 @@ This shared symbol table can be stored in a file (or in a database) to be
 resurrected into a symbol table at runtime.
 
 <script>writeTabs()</script>
+<div class="tabpane C" markdown="1">
+To be determined.
```

Comment:
Maybe better terminology would be "Example not yet implemented." That way we don't indicate that we don't know how to do this in ion-c; we just haven't included an example of it yet. Same below.